docs/aries-cloud-agent-demo.md

```diff
@@ -0,0 +1,241 @@
+<div class="admonition note">
+  <p class="admonition-title">Note</p>
+  <p>This demo is incomplete and will soon be updated.
+</p>
+</div>
+
+Demonstrating an entire decentralised identity scenario involving AEAs and instances of Aries Cloud Agents (ACAs).
+
+## Discussion
+
+This demo corresponds with the one <a href="https://github.com/hyperledger/aries-cloudagent-python/blob/master/demo/README.md" target=_blank>here</a> from <a href="https://github.com/hyperledger/aries-cloudagent-python" target=_blank> aries cloud agent repository </a>. 
+
+There are two AEAs: 
+
+ * Alice_AEA
+ * Faber_AEA 
+
+and two ACAs:
+
+ * Alice_ACA
+ * Faber_ACA
+ 
+Each AEA is connected to its corresponding ACA: Alice_AEA to Alice_ACA and Faber_AEA to Faber_ACA.
+
+The following lists the sequence of interactions between the four agents:
+
+ * Alice_AEA: starts
```

Comment:
A UML diagram would be nice here (see other demos)

Reply:
Yes. What tool do you use?